_shows/73_74/a_resounding_tinkle.md

```diff
@@ -0,0 +1,16 @@
+---
+title: A Resounding Tinkle
+playwright: N F Simpson
```

Comment:
We have this playwright for [One Way Pendulum](https://history.newtheatre.org.uk/years/65_66/one_way_pendulum/) but without a space between the N and F, should be corrected.